apps/prairielearn/package.json

```diff
@@ -245,10 +246,21 @@
     "mocha": "^10.4.0",
     "mocha-steps": "^1.3.0",
     "set-cookie-parser": "^2.6.0",
-    "tsx": "^4.10.2",
+    "tsx": "^4.10.4",
     "typescript": "^5.4.5",
     "typescript-cp": "^0.1.9"
   },
+  "c8": {
+    "reporters": [
+      "html",
+      "text-summary",
```

Comment:
I opted to use `text-summary` instead of `text` because the latter makes it quite hard to review test failures at the end of a run. The expectation is that one will use the HTML view at `coverage/index.html` if they want to see detailed reports.